buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -186,6 +187,10 @@ def _get_build_cache_options(self, builder: Optional[str]) -> dict:
         actual_builder = builder or "default"
         return cache_options if actual_builder in self._cache_builders else {}
 
+    def _log_buildx(self, logs_itr: Iterator[str], platform: str):
+        for log in logs_itr:
+            LOGGER.info(f"[{platform}] {log.strip()}".strip())
```

Comment:
The second strip seems uneccessary.

Reply:
I added the 2nd strip as a personal nitpick. Otherwise when log is blank it puts a space at the end. Not a big deal but my editor made those spaces red and it bothered me. I removed the first strip, since it wasn't needed to accomplish the same thing.